lib/working_hours/computation.rb

```diff
@@ -85,6 +85,26 @@ def advance_to_working_time time, config: nil
       end
     end
 
+    def advance_to_closing_time time, config: nil
+      config ||= wh_config
+      time = in_config_zone(time, config: config).round
+      loop do
+        # skip holidays and weekends
+        while not working_day?(time, config: config)
+          time = (time + 1.day).beginning_of_day
+        end
+        # find next working range after time
+        time_in_day = time.seconds_since_midnight
+        time = time.beginning_of_day
```

Comment:
Ah I see, you properly changed the return and increment so it's fine.
Can you please also add a test for the custom "midnight" handling (24:00) being represented as (23:59:59.99999)
So test with period ending at 24:00 and with current time being midnight or stuff like that.